package.json

```diff
@@ -111,6 +111,7 @@
     "yorkie": "^2.0.0"
   },
   "peerDependencies": {
+    "@dialpad/dialtone": "^7.30.0 || ^8.12.1",
```

Comment:
I think the current version of dialtone vue is not compatible at all with dialtone 7 right?

Reply:
I'd say it's partially compatible, but we need that because product is using both versions and if we don't have that, it'd break the app.

Reply:
oh yeah good point haha